pkg/cmd/run.go

```diff
@@ -73,18 +73,22 @@ doppler run --command "YOUR_COMMAND && YOUR_OTHER_COMMAND"`,
 		localConfig := configuration.LocalConfig(cmd)
 
 		utils.RequireValue("token", localConfig.Token.Value)
-		utils.RequireValue("project", localConfig.EnclaveProject.Value)
-		utils.RequireValue("config", localConfig.EnclaveConfig.Value)
 
 		fallbackPath := ""
+		legacyFallbackPath := ""
 		if cmd.Flags().Changed("fallback") {
 			var err error
 			fallbackPath, err = utils.GetFilePath(cmd.Flag("fallback").Value.String())
 			if err != nil {
 				utils.HandleError(err, "Unable to parse --fallback flag")
 			}
 		} else {
-			fallbackPath = defaultFallbackFile(localConfig.EnclaveProject.Value, localConfig.EnclaveConfig.Value)
+			fallbackPath = defaultFallbackFile(localConfig.Token.Value, localConfig.EnclaveProject.Value, localConfig.EnclaveConfig.Value)
+			// TODO remove this when releasing CLI v4
+			if localConfig.EnclaveProject.Value != "" && localConfig.EnclaveConfig.Value != "" {
+				// save to old path to maintain backwards compatibility
+				legacyFallbackPath = legacyFallbackFile(localConfig.EnclaveProject.Value, localConfig.EnclaveConfig.Value)
```

Comment:
I assume this is on the writing side? Why not read to the new path for these as well?

Reply:
We support reading from the legacy path as well in `readFallbackFile()`